makefiles/pseudomodules.inc.mk

```diff
@@ -52,6 +52,7 @@ PSEUDOMODULES += saul_adc
 PSEUDOMODULES += saul_default
 PSEUDOMODULES += saul_gpio
 PSEUDOMODULES += schedstatistics
+PSEUDOMODULES += openthread
```

Comment:
Looks like this file is sorted alphabetically so this line should just after `newlib_nano`